### PR TITLE
Update UG and usage messages

### LINKED ISSUE #202: [PE-D][Tester B] Unclear Definition of Duplicate Entries
## Describe the Bug

There is no clear specification of what is considered a duplicate for an owner and a pet. From my testing, it seems that owners who share the same IC number are considered duplicates while only pets who share the exact same details are considered duplicates. It would be better if this discrepancy is mentioned in the UG.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: amoschee/ped#4

### LINKED ISSUE #203: [PE-D][Tester C] After adding a link, and opening help, no message explaining how to get back to normal list.
I added a link using link o1 t/p2 

Then I typed help

This got rid of the message reminding me to type list to go back to the original list and for a new user, they may be stuck on this page 

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: rahula1008/ped#7

### LINKED ISSUE #215: [PE-D][Tester A] User Guide photo does not match expected output
In the user guide, it stated that `find pet fluffy megatron returns below.

However, upon typing in `find pet fluffy` I get both a list of owners and pets.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: Teddayz/ped#6

### LINKED ISSUE #248: [PE-D][Tester E] Incorrect screenshot shown for find pet command
The UG states that after executing "find pet fluffy megatron", the UI should only display the matching pets as shown in the screenshot:

However, when I executed the same command in the app, the UI displays both owners and pets.

Steps to reproduce:
1. owner i/S1234567D n/John Doe p/98765432 e/johnd@example.com a/311,
Clementi Ave 2, #02-2
2. pet n/Fluffy s/Dog b/Golden Retriever a/7 x/F
3. pet n/Megatron s/Cat b/Siamese a/3 x/M t/playful
4. find pet fluffy megatron

Suggestion: update the UG to show the correct screenshot

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: yxtan02/ped#14

### LINKED ISSUE #251: [PE-D][Tester E] More screenshots in UG
In the UG, for the sections on adding an owner, adding a pet, edit and sort, more screenshots can be shown to provide a visual representation of the expected behaviour. I believe this would help to improve user experience and verify if the app is working as intended in a quick and efficient manner

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: yxtan02/ped#16

### LINKED ISSUE #261: Ui.png missing
As per guidelines, we need to name Ui.png instead of Ui.jpeg regardless of the filetype we use

### PR DESCRIPTION
**Pull request by clarissatjx**
Fixes #202 
Fixes #203 
Fixes #215 
Related to #227
Related to #241 
Related to #242 
Related to #244 
Fixes #248 
Fixes #251 
Fixes #261
for the following issues, some need updates to DG (hence I didn't link these issues, only mention them). this PR does not include updates to DG yet.

### CODE REVIEW

**Comment by clarissatjx** [comment]
not done yet paiseh, i want to rearrange the ug such that its less cluttered

**Comment by clarissatjx** [comment]
nvm ready for review alr

File: docs/UserGuide.md [hunk 1]
```diff
@@ -97,21 +103,23 @@ Adds a pet owner to PawPatrol.
...
-Examples:
-* `owner i/S1234567D n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25`
+**:exclamation: More on NRIC fields:**<br>
+<ul>
+<li>Due to PDPA regulations, only the last 3 digits and last letter of the NRIC will be shown in the list view.
+<ul><li>The first 4 numbers of the NRIC will be replaced with a `X` to protect the privacy of the owner.</li></ul>
+</ul>
+</div>
 
 > 2 owners may not share the same IC_NUMBER and will be deemed as duplicate
```
Comment (Runus01) [inline docs/UserGuide.md:115]: Should be "deemed as duplicates". I'll update this in a later commit. Will merge in first

**Review by Runus01 — APPROVED** [review]
LGTM! Thanks for working on this!

**Pull request merged**